Docker/AlmaLinux9/Dockerfile-externals

```diff
@@ -0,0 +1,71 @@
+###############################################################################
+#  Repository: ${REPOSITORY}/key4hep-externals
+#  Tag:        ${VERSION}-alma9
+###############################################################################
+
+FROM ghcr.io/key4hep/key4hep-images/alma9:latest
```

Comment:
I'll be making some changes to trim down the images quite significantly and improve how cvmfs is set up in the version with cvmfs. However, they will still be used for building the Key4hep stack so hopefully that won't affect you.